src/sagemaker/session.py

```diff
@@ -4499,9 +4499,19 @@ def get_create_model_package_request(
             "Containers": containers,
             "SupportedContentTypes": content_types,
             "SupportedResponseMIMETypes": response_types,
```

Comment:
I believe these 2 fields are also optional in case of Model registry in APIs. verify once and then update these also.